index.html

```diff
@@ -272,134 +315,188 @@ <h2>Glossary</h2>
+<p><dfn class="lint-ignore export">Shaping</dfn>. Making context-sensitive changes to glyph shapes. Shaping may or may not occur at the same time as context-sensitive positioning of glyphs (such as higher diacritics over tall base characters).</p>
 
-<p><dfn data-lt="syntactic content|Syntactic content" id="def_syntactic_content" class="lint-ignore export">Syntactic content</dfn> is any text in a document format or protocol that belongs to the structure of the format or protocol. This definition includes values that are typically thought of as "markup" but can also include other values, such as the name of a field in an HTTP header. Syntactic content consists of all of the characters that form the <em>structure</em> of a format or protocol. For example, <span class="qchar">&lt;</span> and <span class="qchar">&gt;</span> (as well as the element name and various attributes they surround) are part of the syntactic content in an HTML document. Syntactic content usually is defined by a specification or specifications and includes both the defined, reserved keywords for the given protocol or format as well as string tokens and identifiers that are defined by document authors to form the structure of the document (rather than the "content" of the document).  See also <a class="termref" href="#def_localizable_text">localizable text</a> and <a class="termref" href="#def_natural_language">natural language</a>.</p>
+<p><dfn class="lint-ignore export">Spacing mark</dfn>. Combining characters that consume space along the baseline, either before or after the base character.</p>
+
+<p><dfn class="lint-ignore export">Standalone vowel</dfn>. Vowel sounds that are not immediately preceded by a consonant sound; they typically appear at the beginning or in the middle of a word. In Brahmi-derived scripts standalone vowels are often represented using [=independent vowel=] letters.</p>
+
+<p><dfn class="lint-ignore export" data-lt="supplementary code point">Supplementary characters</dfn>. Beyond the [=Basic Multilingual Plane=] the Unicode character set also contains space for around a million additional code point positions. Characters in this latter range are referred to as supplementary characters. In the UTF-16 encoding, supplementary characters are encoded using a pairs of surrogate characters.</p>
+
+<p><dfn class="lint-ignore" data-lt="surrogate">Surrogate code point</dfn>. <a href="https://www.unicode.org/glossary/#surrogate_code_point">Unicode definition</a>: <q>A Unicode code point in the range U+D800..U+DFFF. Reserved for use by UTF-16, where a pair of surrogate code units (a high surrogate followed by a low surrogate) “stand in” for a [=supplementary code point=].</q></p>
```

Comment:
Should we link to [Infra](https://infra.spec.whatwg.org/#surrogate) here?

Reply:
Added a pointer to INFRA.  Question is whether we should remove the 'export' label.  Our definition provides more information (via Unicode) than the INFRA definition, such as clarifying that the context is UTF-16, which is necessary for things non-HTML.

Reply:
I'm going to leave as is for now, so that we can publish an update of all the stuff we do have ready.